### DISCUSSION TITLE
I tried compiling a statement that prints hello world and found that the generated executable file is too large. Is there a way to reduce the execution volume?

### DISCUSSION THREAD

**Discussion by BestQiang** [traefik/yaegi#1555] [Q&A] [answered]
I tried compiling a statement that prints hello world and found that the generated executable file is too large. Is there a way to reduce the execution volume?

**Comment by mvertes** [comment, marked as answer]
The size can be reduced by not including the full standard library symbols:
```go
package main

import "github.com/traefik/yaegi/interp"

func main() {
  i := interp.New(interp.Options{})
  i.Eval(`println("Hello world")`)
}
```

The executable size is 6M instead of 24M if stdlib is included (`i.Use(stdlib.Symbols)`).

**Reply by BestQiang** [reply]
Can I choose to import only the required packages? If i.Use(stdlib.Symbols) is not used, some packages cannot be imported even if they are used

**Reply by mvertes** [reply]
you have to write your own version of stdlib/stdlib.go , and include only the packages you need.

**Reply by andig** [reply]
That‘s not super simple since it requires an /internal command. Any chance to make that command installable?

**Update** ahh, nvm:

> For a similar purpose with third party packages, see the yaegi extract subcommand,
based on the same code.
